modules/assignment/src/main/resources/assignment.properties

```diff
@@ -27,6 +27,14 @@ http.proxy.host:${http.proxy.host}
 http.proxy.port:${http.proxy.port}
 ruleCache.executor.pool.size:5
 
+#Specify the number of messages to be drained on the event of queue full (executor.max.queue.size).
+#Default value is 10_000
+drain.queue.by=10000
```

Comment:
Agreed @bargenilesh @iizrailevsky I figured that and that is the reason why I said that you should create appropriate bindings in the code with named annotations so that when it needs to be extended, the property values are already available to be used with named annotations.